src/routes.json

```diff
@@ -107,7 +113,7 @@
     },
     {
         "name": "guidelines",
-        "pattern": "^/community_guidelines$",
+        "pattern": "^/community_guidelines/$",
```

Comment:
Whoops, that would be my mistake. Shouldn't there also be a `?` in there too though?

`^/community_guidelines/?$`
instead of
`^/community_guidelines/$`